airflow-core/tests/unit/utils/test_log_handlers.py

```diff
@@ -105,6 +105,7 @@ def test_default_task_logging_setup(self):
         handler = handlers[0]
         assert handler.name == FILE_TASK_HANDLER
 
+    @pytest.mark.xfail(reason="TODO: Needs to be ported over to the new structlog based logging")
```

Comment:
Create an issue for these please

Reply:
Yup, done: https://github.com/apache/airflow/issues/50977